buildbot/master.cfg

```diff
@@ -347,6 +347,145 @@ def make_fifoci_win(type, mode="normal"):
+                   workdir="build/build",
+                   description="testing",
+                   descriptionDone="test",
+                   haltOnFailure=True))
+
+    f.addStep(ShellCommand(command="/build/codesign.sh --deep universal/Dolphin.app",
+                           workdir="build/build",
+                           description="signing",
+                           descriptionDone="sign",
+                           haltOnFailure=True))
+
+    f.addStep(ShellCommand(command="rm -rf dmg.dir && mkdir dmg.dir",
+                           workdir="build/build",
+                           description="creating tmp dir",
+                           descriptionDone="create tmp dir",
+                           haltOnFailure=True,
+                           hideStepIf=StepWasSuccessful))
+
+    f.addStep(ShellCommand(command="cp -R universal/Dolphin.app dmg.dir",
+                           workdir="build/build",
+                           description="creating tmp dir",
+                           descriptionDone="create tmp dir",
+                           haltOnFailure=True,
+                           hideStepIf=StepWasSuccessful))
+
+    f.addStep(ShellCommand(command="cp -R universal/Dolphin\ Updater.app dmg.dir",
+                           workdir="build/build",
+                           description="creating tmp dir",
+                           descriptionDone="create tmp dir",
+                           haltOnFailure=True #,hideStepIf=StepWasSuccessful
+                           ))
+
+    f.addStep(ShellCommand(command=r"if [ -d universal/Dolphin\ Updater.app ]; then /build/codesign.sh --deep universal/Dolphin\ Updater.app; cp -R universal/Dolphin\ Updater.app dmg.dir; fi",
+                            workdir="build/build",
+                            description="signing updater",
+                            descriptionDone="sign updater",
+                            haltOnFailure=True))
+
+    f.addStep(ShellCommand(command=["hdiutil", "create", "dolphin.dmg",
+                                    "-format", "UDBZ",
+                                    "-fs", "HFS+", # Needed for 7-Zip support
+                                    "-srcfolder", "dmg.dir", "-ov",
+                                    "-volname", "Dolphin"],
+                                    #"-volname", "dolphin.dmg"],
+                           workdir="build/build",
+                           logEnviron=False,
+                           description="packaging",
+                           descriptionDone="package"))
+
+    f.addStep(ShellCommand(command="/build/codesign.sh --deep dolphin.dmg",
+                           workdir="build/build",
+                           description="signing dmg",
+                           descriptionDone="sign dmg",
+                           haltOnFailure=True))
+
+    if mode == "normal":
+        master_filename = WithProperties("/srv/http/dl/builds/dolphin-%s-%s.dmg", "branchname", "shortrev")
```

Comment:
All these paths and URLs need to be different from the non-universal macOS builds to prevent conflicts.

Reply:
Done